tools/lib/provision.py

```diff
@@ -109,13 +109,13 @@
 if (not is_rhel_based) and (not os.path.exists("/usr/bin/lsb_release")):
     run_as_root(["apt-get", "install", "-y", "lsb-release"])
 
-distro_info = parse_lsb_release()
-vendor = distro_info['DISTRIB_ID']
-codename = distro_info['DISTRIB_CODENAME']
+distro_info = parse_os_release()
+vendor = distro_info['ID']
+codename = distro_info['VERSION_ID']
```

Comment:
Let’s rename this variable to `version` or `os_version` or something.

Reply:
Several scripts are still using `lsb_release`, which indeed has `codename` as a field. There is also a puppet script that uses `os_codename`. So, for consistency, either use `codename` across all the files, or use `os_version` across the files while also removing the use of `lsb_release` program call.

Reply:
9th commit.